### DISCUSSION TITLE
What other achievements other than besides YOLO, Pull shark, and Galaxy Brain?

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#11] [Q&A] [answered]
Anyone knows?

**Comment by ale-lit** [comment]
So far, only those described here are known: https://github.com/Schweinepriester/github-profile-achievements

Perhaps there will be new achievements later.

**Comment by dcai** [comment]
Public Sponsor

**Comment by melvincwng** [comment, marked as answer]
If you are referring to the new achievements, other than those 3, we do not know what other achievements there are (at the moment).

If you are referring to the old achievements, there's Arctic Code Vault Contributor, Public Sponsor, and Mars 2020 Contributor. Only Public Sponsor is available to get now.

**Comment by jazzybruno** [comment]
Apart from the listed above we also have the artic vault and mars helicopter mission but thise cant be earned unless you contributed si the time  for earning them has expired

**Comment by jazzybruno** [comment]
But you can also get the public sponsor achievement for sponsoring the open source work on github

**Comment by ItzJustNick** [comment]
I think you need to answer 2 disscusions and then become the accepted answer on both to get the achivement. Hope this helps.
